app/src/main/res/values/strings.xml

```diff
@@ -305,7 +305,7 @@
     <string name="compose_active_account_description">Posting with account %1$s</string>
 
     <string name="error_failed_set_caption">Failed to set caption</string>
-    <string name="hint_describe_for_visually_impaired">Describe for visually impaired</string>
+    <string name="hint_describe_for_visually_impaired">Describe for visually impaired (%d character limit)</string>
```

Comment:
you should change all other languages as well, or delete the  string in other languages